build.sbt

```diff
@@ -126,7 +126,7 @@ lazy val commonSettings = Def.settings(
 
 lazy val root = project
   .in(file("."))
-  .enablePlugins(PlayRootProject, ScalaJSPlugin, PlayNoPublish)
```

Comment:
As `PlayNoPublish` was removed, we need to add the settings manually. I think `publish / skip := true` should be enough (as [documented here](https://www.scala-sbt.org/1.x/docs/Publishing.html#Skip+publishing)) but maybe re-add `publishTo: None` as well.

Reply:
`publish / skip := true` is what we use in other projects (eg https://github.com/playframework/play-grpc/blob/master/project/Publish.scala#L13).

I've ran `show publishTo` with this setup and the `root` project reports no target (which is what I think we wanted).

How about we merge and we revisit this on the next release?